src/coreclr/jit/compiler.cpp

```diff
@@ -9201,6 +9201,14 @@ XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
 XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
 */
 
+#ifndef DBGFUNC
+#ifdef _MSC_VER
+#define DBGFUNC __declspec(dllexport)
```

Comment:
We don't use `__declspec(dllexport)` elsewhere; we use the ClrJit.exports and ClrJit.PAL.exports files.

I notice that some (but not all?) such functions are marked  `extern "C"`. Also seems weird that `DllMain` isn't in the ClrJit.exports file but maybe it's special?

Reply:
I checked and MSVC just combines the exports, so both these functions and the ones from the file end up exported. Since these are only exported in debug builds I think it's better to keep it out of the file.

We don't need `extern "C"` on these, the debugger has no problem with demangling them.